client/my-sites/themes/theme-showcase.jsx

```diff
@@ -378,9 +386,11 @@ const mapStateToProps = ( state, { siteId, filter, tier, vertical } ) => {
 	const currentThemeId = getActiveTheme( state, siteId );
 	const currentTheme = getCanonicalTheme( state, siteId, currentThemeId );
 	return {
+		canUploadThemes: hasActiveSiteFeature( state, siteId, FEATURE_UPLOAD_THEMES ),
```

Comment:
Should this be `FEATURE_INSTALL_THEMES`?

Reply:
Installing a theme is the same as uploading it, isn't it? In any case, I think this should check a new `FEATURE_ACTIVATE_THEMES` constant, since My Themes only allows users to activate themes that are already installed in the site.

Reply:
Did some more research and it looks like `install_themes` lets users install themes from the Theme Directory while `upload_themes` only enables the theme uploader in wp-admin.

So, yeah, I think I agree, checking for `switch_themes` makes sense since they're not able to install themes here.

Reply:
Oh right, I guess it would depend on how we sell the Business plan.

If we say that users are only allowed to **use** custom themes on the Business plan, I think we shouldn't differentiate between install/upload/switch, and instead create a more generic feature/capability that includes the whole management of custom themes. That new generic feature/capability is the one we should check everywhere to prevent users from installing/uploading/activating custom themes if the site doesn't have the required upgrade.

On the other hand, if we say that users cannot **upload** custom themes on the Business plan but they are free to **activate** the ones they uploaded while the site had the Business plan active, then we should check `switch_themes` which would keep My Themes available even if the site doesn't have the required upgrade.